addons/website_event_track/controllers/event.py

```diff
@@ -1,41 +1,19 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    Copyright (C) 2013-Today OpenERP SA (<http://www.openerp.com>).
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-import collections
 
+import collections
 import datetime
-import re
-
 import pytz
+import re
 
-import openerp
-import openerp.tools
 from openerp.addons.web import http
 from openerp.addons.web.http import request
+from openerp.tools import html_escape, DEFAULT_SERVER_DATETIME_FORMAT as DTF
 
 class website_event(http.Controller):
     @http.route(['''/event/<model("event.event"):event>/track/<model("event.track", "[('event_id','=',event[0])]"):track>'''], type='http', auth="public", website=True)
     def event_track_view(self, event, track, **post):
-        track_obj = request.registry.get('event.track')
-        track = track_obj.browse(request.cr, openerp.SUPERUSER_ID, track.id, context=request.context)
-        values = { 'track': track, 'event': track.event_id, 'main_object': track }
+        track = track.sudo()
+        values = { 'track': track, 'event': track.event_id }
```

Comment:
main_object use for create page title  https://github.com/odoo/odoo/blob/master/addons/website/views/website_templates.xml#L66
if you want to remove main_object from here then set main_object as track in template